Source/RimConnection/API/CommandOption.cs

```diff
@@ -1,18 +1,21 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using RestSharp;
+using Verse;
 
 #pragma warning disable IDE1006 // Naming Styles
 namespace RimConnection
 {
-    public class CommandOption : ICloneable
+    public class CommandOption : ICloneable, IExposable
     {
-        public string actionHash { get; set; }
-        public int localCooldownMs { get; set; }
-        public int globalCooldownMs { get; set; }
-        public int costSilverStore { get; set; }
+        private string _actionHash;
+        private int _localCooldownMs;
+        private int _globalCooldownMs;
+        private int _costSilverStore;
+
+        public string actionHash { get => _actionHash; set => _actionHash = value; }
+        public int localCooldownMs { get => _localCooldownMs; set => _localCooldownMs = value; }
+        public int globalCooldownMs { get => _globalCooldownMs; set => _globalCooldownMs = value; }
+        public int costSilverStore { get => _costSilverStore; set => _costSilverStore = value; }
```

Comment:
Can we include the name here? I think some people want to be able to edit these files so knowing what they are would be helpful to them